kyaml/openapi/README.md

```diff
@@ -22,22 +22,26 @@ make nuke
 
 The compiled-in schema version should maximize API availability with respect to all actively supported Kubernetes versions. For example, while 1.20, 1.21 and 1.22 are the actively supported versions, 1.21 is the best choice. This is because 1.21 introduces at least one new API and does not remove any, while 1.22 removes a large set of long-deprecated APIs that are still supported in 1.20/1.21.
 
-### Update the built-in schema to a new version
+### Generating additional schema
 
-In the Makefile in this directory, update the `API_VERSION` to your desired version.
+If you'd like to change the default schema version, then in the Makefile in this directory, update the `API_VERSION` to your desired version.
 
 You may need to update the version of Kind these scripts use by changing `KIND_VERSION` in the Makefile in this directory. You can find compatibility information in the [kind release notes](https://github.com/kubernetes-sigs/kind/releases).
 
-In this directory, fetch the openapi schema and generate the 
-corresponding swagger.go for the kubernetes api: 
+In this directory, fetch the openapi schema, generate the
+corresponding swagger.go for the kubernetes api, and update `kubernetesapi/openapiinfo.go`:
 
 ```
-make all
+make kustomizationapi/swagger.go
```

Comment:
I changed this because `make all` now seems like it's trying to run the `make all` from the top level kustomize directory, not super sure why that happens. Running these three make commands is what we want so I just put them all here.

Reply:
🤔 It mysteriously went away....... I recall triple checking that I was in the right directory, might have been something weird with my env. I'll change this back.